.github/workflows/build.yaml

```diff
@@ -158,6 +158,18 @@ jobs:
         name: "update__${{ github.ref_name }}_(${{ steps.vars.outputs.sha_short }})"
         path: ./dist/*
 
+    - name: Store generated files in cache
```

Comment:
Unless you modified the files, there is no need to push them to (another) cache!